i18n/pt-BR.json

```diff
@@ -3076,6 +3144,10 @@
     "translation": "Inválido máxima tentativas de login em configurações de serviço.  Deve ser um número positivo."
   },
   {
+    "id": "model.config.is_valid.max_burst.app_error",
+    "translation": "Max burst must be greater than zero."
```

Comment:
yeap, probably the translation wasn't in the translations server when I submitted the PR, the same goes for the rest of the languages